packages/help-center/src/hooks/use-action-hooks.ts

```diff
@@ -9,29 +9,27 @@ export const useActionHooks = () => {
 	const { setShowHelpCenter, setShowSupportDoc, setNavigateToRoute } =
 		useDispatch( 'automattic/help-center' );
 	const queryParams = new URLSearchParams( window.location.search );
+	const helpCenterParam = queryParams.get( 'help-center' ) || '';
 
 	const actionHooks = [
 		/**
-		 * Open to the support doc for the Subscribe block.
+		 * Open to a specific support doc.
 		 */
 		{
 			condition() {
-				return queryParams.get( 'help-center' ) === 'subscribe-block';
+				return isThisASupportArticleLink( helpCenterParam );
```

Comment:
Just checking this, and I believe we can't remove the subscribe-block action hook, since it is used in launchpad